action.yml

```
@@ -127,13 +169,19 @@ runs:
           HOSTNAME: ${{ inputs.hostname }}
           TAILSCALE_AUTHKEY: ${{ inputs.authkey }}
           TIMEOUT: ${{ inputs.timeout }}
-          TS_EXPERIMENT_OAUTH_AUTHKEY: true
         run: |
           if [ -z "${HOSTNAME}" ]; then
-            HOSTNAME="github-$(cat /etc/hostname)"
+            if [ "{{ runner.os }}" == "Linux" ]; then
+              HOSTNAME="github-$(cat /etc/hostname)"
+            elif [ "{{ runner.os }}" == "Windows" ]; then
+              HOSTNAME="github-$COMPUTERNAME"
+            fi
```

missing `$`s in front of the braces
```suggestion
            if [ "${{ runner.os }}" == "Linux" ]; then
              HOSTNAME="github-$(cat /etc/hostname)"
            elif [ "${{ runner.os }}" == "Windows" ]; then
              HOSTNAME="github-$COMPUTERNAME"
            fi
```